CHANGES.md

```diff
@@ -1,6 +1,427 @@
+* UTF-8 support no longer depends on the `LANG` environment variable  
+  (see https://chapel-lang.org/docs/1.26/usingchapel/chplenv.html#character-set)
+* removed support for the deprecated `CHPL_REGEXP` environment variable
+
+Syntactic / Naming Changes
+--------------------------
+* extended the set of statement types to which `try`/`try!` can be applied
+
+Semantic Changes / Changes to Chapel Language
+---------------------------------------------
+* modules from files listed on the command-line are now always initialized  
+  (see https://chapel-lang.org/docs/1.26/language/spec/modules.html#module-initialization)
+* enabled implicit conversions from non-nilable to nilable for type arguments  
+  (see https://chapel-lang.org/docs/1.26/language/spec/procedures.html#legal-argument-mapping)
+* `.indices` queries on rectangular arrays now return a local domain of indices  
+  (see https://chapel-lang.org/docs/1.26/language/spec/arrays.html#ChapelArray.indices)
+* `.size`/`.shape` queries on ranges, domains, arrays now return `int`s  
+  (see https://chapel-lang.org/docs/1.26/builtins/ChapelRange.html#ChapelRange.range.size)
+* stopped auto-creating `hash()` methods for records w/ custom `==`/`!=` ops  
+  (see https://chapel-lang.org/docs/1.26/language/spec/records.html#hashing-a-record)
+* made it an error to overload return intents for the `enterThis()` method  
+  (see https://chapel-lang.org/docs/1.26/language/spec/statements.html#the-manage-statement)
+
+New Features
+------------
+* added support for listing operators in `only`, `except`, and `import`  
+  (see https://chapel-lang.org/docs/1.26/language/spec/modules.html#using-modules  
+   and https://chapel-lang.org/docs/1.26/language/spec/modules.html#importing-modules)
+* added an `unsafeAssign()` method and context manager to domains  
+  (see https://chapel-lang.org/docs/1.26/language/spec/domains.html#ChapelDomain.unsafeAssignManager)
+* added support for renaming `extern type` declarations  
+  (e.g., `extern "c_name" type chpl_name = ...;`  
+   see also https://chapel-lang.org/docs/1.26/language/spec/interoperability.html#referring-to-external-c-types)
+
+Feature Improvements
+--------------------
+* extended hash-based collections to support values of range/array type
+
+Deprecated / Unstable / Removed Language Features
+-------------------------------------------------
+* deprecated support for arrays over bracket-less anonymous associative arrays  
+  (e.g., in `var A: ["red", "green", "blue"] real;`, brackets are now required)
+* deprecated support for `ChapelEnv.CHPL_AUX_FILESYS`  
+  (see https://chapel-lang.org/docs/1.26/modules/standard/ChplConfig.html#ChplConfig.CHPL_AUX_FILESYS)
+* removed deprecated support for `use <Mod> except *;`
+* removed deprecated support for operators declared using the `proc` keyword
+* removed all vestiges of the `atomic` statement
+
+Namespace Changes
+-----------------
+
```

Comment:
I would prefer a N/A here if there haven't been any namespace changes.

Reply:
I always GC any blank entries once we're done adding new content.  I was still waiting on additional entries after stopping on Friday.